modules/output.py

```
@@ -0,0 +1,6 @@
+from modules.util import get_output_folder
+from modules.auth import get_current_user
+
+def save_image(image, filename):
+    output_path = get_output_folder()
+    image.save(os.path.join(output_path, filename))
```

Ruff (0.8.2)

Undefined name `os`

(F821)